runtime/rasdump/dmpagent.c

```diff
@@ -2757,7 +2760,38 @@ runDumpAgent(struct J9JavaVM *vm, J9RASdumpAgent * agent, J9RASdumpContext * con
 		if (gotExclusive || (agent->dumpFn != doHeapDump)) {
 			agent->prepState = *state;
 			TRIGGER_J9HOOK_VM_DUMP_START(vm->hookInterface, vm->internalVMFunctions->currentVMThread(vm), label, detail);
+#if defined(J9ZTPF)
+			struct cujvm_dmpagent_prehook_input preHookInputParms;
+			struct cujvm_dmpagent_prehook_output preHookOutputParms;
+			struct cujvm_dmpagent_posthook_input postHookInputParms;
+			struct cujvm_dmpagent_posthook_output postHookOutputParms;
```

Comment:
This isn't legal C code: declarations must be at the beginning of a block.
I think it makes sense to put this before the hook above anyway so the updated `label` can be shared.